homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,11 @@
+import app from "./app.js";
+
+const PORT = 3000;
+
+app.listen(PORT, (error) =>{
+  if(!error)
+      console.log("Server is Successfully Running, and App is listening on port "+ PORT)
+  else 
+      console.log("Error occurred, server can't start", error);
+  }
```

Comment:
`listen`'s callback method doesn't have any parameters.
```suggestion
app.listen(PORT, () =>{
    console.log("Server is Successfully Running, and App is listening on port "+ PORT)
  }
```